build-logic-commons/build-platform/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ val kotlinVersion = providers.gradleProperty("buildKotlinVersion")
 dependencies {
     constraints {
         api("org.gradle.guides:gradle-guides-plugin:0.23")
-        api("org.apache.ant:ant:1.10.14") // Bump the version brought in transitively by gradle-guides-plugin
+        api("org.apache.ant:ant:1.10.15") // Bump the version brought in transitively by gradle-guides-plugin
```

Comment:
🤔 do we need to upgrade it here?

Reply:
We needed a different version because there is a high severity CVE in the one coming from gradle-guides-plugin. It's easier to just keep these in sync, but we could change this to just require something newer than the version with the CVE.

I don't remember why gradle-guides-plugin even needs Ant and since we use so little of gradle-guides-plugin now, we can probably get rid of the dependency.